java-persistence/src/main/java/com/whitecloak/training/inventory/service/UserService.java

```diff
@@ -0,0 +1,12 @@
+package com.whitecloak.training.inventory.service;
+
+import com.whitecloak.training.inventory.controller.request.UserForm;
+import com.whitecloak.training.inventory.controller.response.UserResource;
+import com.whitecloak.training.inventory.persistence.entity.UserEntity;
+
+public interface UserService {
+    UserResource create(UserForm form);
+
+    UserEntity getUser(Long id);
```

Comment:
Entity has leaked outside the gateways. Remember to contain entities only in the gateways.